filebeat/docs/modules/zookeeper.asciidoc

```diff
@@ -0,0 +1,77 @@
+////
+This file is generated! See scripts/docs_collector.py
+////
+
+[[filebeat-module-zookeeper]]
+:modulename: zookeeper
+:has-dashboards: true
```

Comment:
It seems that no dashboards are included, please review this.

Reply:
needed to run `mage fmt update`. done